flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/KeyedStream.scala

```diff
@@ -183,7 +183,10 @@ class KeyedStream[T, K](javaStream: KeyedJavaStream[T, K]) extends DataStream[T]
    * Creates a new [[DataStream]] by folding the elements of this DataStream
    * using an associative fold function and an initial value. An independent 
    * aggregate is kept per key.
+   *
+   * @deprecated will be removed in a future version
    */
+  @Deprecated
```

Comment:
I don't know, actually. It seems actual (random) code on GitHub only has it in the annotation: https://github.com/ReactiveX/RxScala/blob/0.x/src/main/scala/rx/lang/scala/Observable.scala#L4876